packaging/android/build_and_publish.sh

```diff
@@ -30,7 +30,18 @@ fi
 # Build and publish the apk
 cd $android_dir
 
+RN_DIR="$rn_dir" $client_dir/packaging/manage_react_native_packager.sh &
```

Comment:
fyi, don't need to change anything in this PR but always a good idea to always wrap anything with $vars in it with quotes..

e.g. "$client_dir/.." 

just as a habit...